SayTheirNames/Source/View/Home/Person/PersonCell.swift

```diff
@@ -108,6 +110,26 @@ final class PersonCell: UICollectionViewCell {
         containerStack.fillSuperview()
     }
 
+    override func didMoveToSuperview() {
+        super.didMoveToSuperview()
+        
+        styleLabels()
+    }
+    
+    private func styleLabels() {
+
+        nameLabel.font = UIFont.STNDynamic.cardTitle
+        dateOfIncidentLabel.font = UIFont.STNDynamic.cardSubitle
+    }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        
+        if traitCollection.preferredContentSizeCategory != previousTraitCollection?.preferredContentSizeCategory {
```

Comment:
feels like we don't need this bit 🤔  

because fonts in our extension are not self-resizing by intention (then UIs don't need to be so dynamic). users usually don't change text settings on the fly, only developers do :) 
if we wanted fonts to self-resize, there's a way to create fonts that change by themselves without us writing extra code to react to change (using font metrics)

Reply:
If I take out the call from traitCollectionDidChange and test on device, then the text size is not updated until the app is restarted.  For most users, that means waiting until either 1) the app is quit in the background after they've not used it for a while or 2) the phone is restarted.  Tech-save people force-quit their apps.  Most typical users don't, or wouldn't necessarily think of it in the moment.

I'm thinking of 2 scenarios in particular:

1) Someone downloads STN for the first time and launches it to discover that the text size is too small for them.  They go to the system preferences and try to increase the type size.  They return to STN and see that nothing has changed.  They immediately decide the app isn't that useful if they have to squint to read it.

2) Two friends are talking about BLM in the news.  One remembers reading something in STN and loads the story and hands her phone to the other, who is a slightly older person with poorer eyesight. The second friend asks the first to make the screen more readable.  The first tries, but the text size doesn't change. Rather than try to troubleshoot on the spot, they give up.

I see this as an accessibility issue and a usability issue.  I'd politely ask that you reconsider.

Reply:
It definitely couldn't hurt to have an app be more accessible, but i think anyone that is familiar with quickly switching font sizes on iphone would also be familiar with killing apps as well. 

Overall, what i would prefer to see here is that we replace the current STN font code with the computed code so we can refactor in place.

Reply:
>> Overall, what i would prefer to see here is that we replace the current STN font code with the computed code so we can refactor in place.
 
Are you saying replace the properties in STN with the ones in STNDynamic?  I'd definitely like to get there eventually, but didn't want to break other people's code for now with the rate that PRs are coming in. I think this branch is already about 20 commits behind...

>> ...but i think anyone that is familiar with quickly switching font sizes on iphone would also be familiar with killing apps as well.

When's the last time you helped a teacher over 50 with their phone? They tend to see these kinds of features in isolation, not necessarily understand how to use them in combination.  Not to pick on educators, I was one myself until last year...

Reply:
Lets remove `STNDynamic` and refactor the code in `STN` to be dynamic
```
static let title = UIFont.dynamicCustomFont(
        fontName: FontName.karlaBold.rawValue,
        fontName: FontName.karlaBold.rawValue,
        textStyle: .title1)
        textStyle: .title1)
```
to
```
static var title: UIFont { UIFont.dynamicCustomFont(
        fontName: FontName.karlaBold.rawValue,
        textStyle: .title1) }
```

Reply:
easy enough. done